### PR TITLE
Remove remote_markdown completely

### PR DESCRIPTION
**Pull request by dshevtsov**
## Purpose of this pull request

This pull request removes a liquid tag `remote_markdown` completely from the project. Each usage was replaced with a hyperlink to the actual code.

## Affected pages

Removing the topics that were entirely consisted of the content included by `remote_markdown`:
   - <https://devdocs.magento.com/guides/v2.2/mrg/ce/Analytics.html>
   - <https://devdocs.magento.com/guides/v2.2/mrg/ce/CatalogAnalytics.html>
   - <https://devdocs.magento.com/guides/v2.2/mrg/ce/CustomerAnalytics.html>
   - <https://devdocs.magento.com/guides/v2.2/mrg/ce/QuoteAnalytics.html>
   - <https://devdocs.magento.com/guides/v2.2/mrg/ce/ReviewAnalytics.html>
   - <https://devdocs.magento.com/guides/v2.2/mrg/ce/SalesAnalytics.html>
   - <https://devdocs.magento.com/guides/v2.2/mrg/ce/WishlistAnalytics.html>

Links to the topics replaced with links to original READMEs in the [magento/magento2](https://github.com/magento/magento2) repository.

Editing topics that contained any use of `remote_markdown`:
   - guides/v2.2/mrg/ce/Sales.md
   - guides/v2.2/mtf/mtf_entities/mtf_constraint.md
   - guides/v2.2/mtf/mtf_entities/mtf_dataset.md
   - guides/v2.2/mtf/mtf_entities/mtf_handler.md
   - guides/v2.2/mtf/mtf_entities/mtf_scenariotest.md
   - guides/v2.2/test/integration/annotations/magento-data-fixture.md
   - guides/v2.3/test/integration/annotations/magento-data-fixture.md

### CODE REVIEW

File: guides/v2.2/mtf/configuration.md [hunk 1]
```diff
@@ -10,14 +10,7 @@ The Functional Testing Framework configuration settings are located in two [XML]
...
 
-The `<magento2_root_dir>/dev/tests/functional/phpunit.xml` contains PHPUnit configuration settings. Blocks of settings are described in [PHPUnit documentation], except `<php>` that contains parameters described in the following table.
-
-{% collapsible Show/hide the default phpunit.xml code %}
-
-```xml
-{% remote_markdown  https://raw.githubusercontent.com/magento/magento2/2.2/dev/tests/functional/phpunit.xml.dist %}
-```
-{% endcollapsible %}
+The `<magento2_root_dir>/dev/tests/functional/phpunit.xml` contains PHPUnit configuration settings. Blocks of settings are described in [PHPUnit documentation], except `<php>` that contains parameters described in the following table. See the default phpunit.xml code at [`dev/tests/functional/phpunit.xml.dist`][].
```
Comment (keharper) [inline guides/v2.2/mtf/configuration.md]:
```suggestion
The `<magento2_root_dir>/dev/tests/functional/phpunit.xml` contains PHPUnit configuration settings. Blocks of settings are described in the [PHPUnit documentation], except `<php>`, which contains parameters described in the following table. See the default phpunit.xml code at [`dev/tests/functional/phpunit.xml.dist`][].
```

File: guides/v2.2/mtf/configuration.md [hunk 2]
```diff
@@ -38,33 +31,21 @@ The `<magento2_root_dir>/dev/tests/functional/phpunit.xml` contains PHPUnit conf
...
 
-{% collapsible in the Framework %}
-
-The file is located in `<magento2_root_dir>dev/tests/functional/vendor/magento/mtf/etc`
+**In the Framework:**
 
-```xml
-{% remote_markdown https://raw.githubusercontent.com/magento/mtf/develop/etc/config.xml.dist%}
-```
+The file is located in `<magento2_root_dir>/dev/tests/functional/vendor/magento/mtf/etc`. See default configuration at [`etc/config.xml.dist`][].
```
Comment (keharper) [inline guides/v2.2/mtf/configuration.md]:
```suggestion
The file is located in `<magento2_root_dir>/dev/tests/functional/vendor/magento/mtf/etc`. See the default configuration at [`etc/config.xml.dist`][].
```

File: guides/v2.2/mtf/configuration.md [hunk 3]
```diff
@@ -38,33 +31,21 @@ The `<magento2_root_dir>/dev/tests/functional/phpunit.xml` contains PHPUnit conf
...
+**In Magento functional tests:**
 
-{% collapsible in Magento functional tests %}
-
-The file is located in `<magento2_root_dir>dev/tests/functional/`
-
-```xml
-{% remote_markdown https://raw.githubusercontent.com/magento/magento2/2.2/dev/tests/functional/etc/config.xml.dist %}
-```
+The file is located in `<magento2_root_dir>/dev/tests/functional/`. Default configuration is in [`dev/tests/functional/etc/config.xml.dist`][].
```
Comment (keharper) [inline guides/v2.2/mtf/configuration.md]:
```suggestion
The file is located in `<magento2_root_dir>/dev/tests/functional/`. The default configuration is in [`dev/tests/functional/etc/config.xml.dist`][].
```

File: guides/v2.2/mtf/mtf_entities/mtf_handler.md [hunk 4]
```diff
@@ -249,11 +245,7 @@ Let's create a cURL handler that creates a new widget.
   }
   ```
 
-*  Create [`di.xml`](#mtf_handler_di) in the `etc/curl` directory of the Magento_Widget module.
-
-```xml
-{% remote_markdown https://raw.githubusercontent.com/magento/magento2/2.2/dev/tests/functional/tests/app/Magento/Widget/Test/etc/curl/di.xml %}
-```
+*  Create [`di.xml`](#mtf_handler_di) in the `etc/curl` directory of the Magento_Widget module ([`dev/tests/functional/tests/app/Magento/Widget/Test/etc/curl/di.xml`][]).
```
Comment (keharper) [inline guides/v2.2/mtf/mtf_entities/mtf_handler.md]:
```suggestion
*  Create a [`di.xml`](#mtf_handler_di) file in the `etc/curl` directory of the Magento_Widget module ([`dev/tests/functional/tests/app/Magento/Widget/Test/etc/curl/di.xml`][]).
```

**Comment by dshevtsov** [comment]
running tests

**Comment by dshevtsov** [comment]
running tests

**Comment by dshevtsov** [comment]
running tests

**Pull request merged**